enhancements/update/channel-rename-generally-available.md

```diff
@@ -0,0 +1,628 @@
+
+#### Documentation
+
+The current docs are a bit murkey on updates between minor versions.
+For example, 4.15 web-console docs talk about using the 4.15 web-console to update, and thus presumably cover both 4.15-to-4.15 and 4.15-to-4.16.
+4.16 command-line docs talk about using the 4.16 `oc`, and thus presumably cover both 4.15-to-4.16 and 4.16-to-4.16.
+EUS docs are very coy about which versions they're talking about, using placeholders to avoid having to commit.
+Mixing this preexisting complication in with this enhancement's renaming seems like it could be tricky.
+
+Luckily, the channel docs themselves only talk about their particular 4.y (e.g. [4.14's channel docs][channel-docs] only talk about `*-4.14`), so that portion should be straightforward.
+[This ticket][OTA-1220] is exploring how OCP 4.15 docs should cover the new channels.
+
+On top of OCP docs, there is also [ARO docs exposure](#aro).
+And [some ROSA/OSD references to channel group names have leaked into docs](#managed), and may need updating if [Service Delivery decides to rename channel groups](#rename-managed-channel-groups).
+
+There may also be some KCS that reference the existing channel naming paterns, and their maintainers may not be watching closely enough to notice this pivot and remember to update the KCSs (although this seems like a generic risk to all KCSs, and a reason to prefer managed docs over KCSs for any advice that needs long-term maintainence).
+
+#### Installer
+
+No installer changes are expected for 4.15.
+
+[In 4.16, the installer is expected to bump][OTA-1219] their manually-maintained [default channel][installer-default-channel] to use the `ga-4.y` pattern (alternatively, they could [pivot to `fleet-approved-4.y`](#installer-default-channel)).
+
+#### Web-console
+
+As described [here](#historical-context), the web-console is mostly out of the business of hard-coding channel information.
+However, they do have [placeholder and help text in the channel model][web-console-channel-modal] that would need updating.
+Both are only presented when the ClusterVersion's `status.desired.channels` is empty, which limits exposure to clusters which have already picked an invalid channel or cleared their channel, or when the update service access is failing.
+In the absence of this bump, they could mislead cluster administrators into setting old-pattern channel names, and the admins would have to follow the link to [channel docs][channel-docs] to discover appropriate values for their 4.y release.
+
+#### Cluster-version operator
+
+The cluster-version operator could be taught to alert on deprecated channel names.
+Although currently it is not clear if the old naming patterns will be deprecated, or remain GA forever, more in [discontinuing the aliases](#discontinuing-the-aliases).
+
+#### Continuous integration
```

Comment:
I can confirm these findings. It _seems_ that the risk from CI point of view is low. I can't exclude however some situations (especially legacy, historic etc) that will occur once the change is introduced. It won't stop me from giving LGTM from our POV though.